lighthouse-cli/test/smokehouse/test-definitions/perf/expectations.js

```diff
@@ -354,4 +354,31 @@ module.exports = [
       },
     },
   },
+  {
+    lhr: {
```

Comment:
How do you feel about a new category like `perf-serial` and move every test which needs to be run serially in that group?

Reply:
> How do you feel about a new category like `perf-serial` and move every test which needs to be run serially in that group?

to me it makes more sense to keep the metrics tests as `perf` (since they're the core of the perf category) and split these perf opportunities/diagnostics into their own name (and `yarn smoke perf-serial` is a lot more awkward to type :), but I don't feel super strongly about it.

Reply:
Ok I think I will open an issue for this and tackle in a separate PR.